exporter/dynatraceexporter/README.md

```diff
@@ -261,6 +263,45 @@ User should calculate this as `num_seconds * requests_per_second` where:
 
 Default: `5000`
 
+### resource_to_telemetry_conversion (Optional)
+
+When `resource_to_telemetry_conversion.enabled` is set to `true`, all resource
+attributes will be included as metric attributes.
+
+Default: `false`
+
+> :warning: **Please note** that the Dynatrace API has a limit of `50` attributes
+> per metric and any metric which exceeds this limit will be dropped.
+
+If you think you might exceed this limit, you should use the
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+ to remove any resource attributes you don't want converted.
```

Comment:
In your example, you don't specify the ones you drop, but rather supply a filter for which ones you keep, so maybe:
```suggestion
to apply a filter, so only a select subset of your resource attributes are converted.
```